### PR TITLE
Hash_code calculation with Vulnerability Ids

### PR DESCRIPTION
**Pull request by StefanFl**
With this PR the calculation of hash_codes gets changed from `cve` to `vulnerability_ids`.

### CODE REVIEW

File: dojo/tools/cargo_audit/parser.py [hunk 1]
```diff
@@ -39,9 +39,9 @@ def get_findings(self, filename, test):
                 date = advisory.get('date')
 
                 if len(advisory.get('aliases')) != 0:
-                    cve = advisory.get('aliases')[0]
+                    vulnerability_id = advisory.get('aliases')[0]
```
Comment (damiencarol) [inline dojo/tools/cargo_audit/parser.py:42]: `aliases` attribute seems to be an array, could we use it as-is?

Comment (StefanFl) [reply dojo/tools/cargo_audit/parser.py:42]: What do you thing of a larger change: The `id` of the finding in the file is something like `RUSTSEC-2020-0091` which can point to this URL: https://rustsec.org/advisories/RUSTSEC-2020-0091, see https://github.com/DefectDojo/django-DefectDojo/blob/fef0d194ff985358be8e7ebbd4c13a02b9c5a539/unittests/scans/cargo_audit/many_findings.json#L26. 

I think we should use the id and the list of aliases as vulnerability ids.

Comment (damiencarol) [reply dojo/tools/cargo_audit/parser.py:42]: I agree. now we need to make it real :D

Comment (StefanFl) [reply dojo/tools/cargo_audit/parser.py]: Done

**Review by damiencarol — APPROVED** [review]

**Review by Maffooch — APPROVED** [review]

**Pull request merged**